src/dune_engine/action_ast.ml

```diff
@@ -1,10 +1,7 @@
 open! Stdune
 open Import
 open Dune_lang.Decoder
-
-let () =
-  let module M = Stanza in
-  ()
```

Comment:
What was this about?

Reply:
Probably to force ocamldep's behavior and introduce a dependecy on stanza.ml. I recall we had to do ugly hacks like that the old bootstrap procedure.

Reply:
Interesting. Thanks!